rust-runtime/aws-smithy-http-server/examples/Makefile

```diff
@@ -0,0 +1,27 @@
+SRC_DIR := $(shell git rev-parse --show-toplevel)
+CUR_DIR := $(shell pwd)
+GRADLE := $(SRC_DIR)/gradlew
+SERVER_SDK_DST := $(CUR_DIR)/pokemon-service-sdk
+CLIENT_SDK_DST := $(CUR_DIR)/pokemon-service-client
+SERVER_SDK_SRC := $(SRC_DIR)/codegen-server-test/build/smithyprojections/codegen-server-test/pokemon-service-sdk/rust-server-codegen
+CLIENT_SDK_SRC := $(SRC_DIR)/codegen-test/build/smithyprojections/codegen-test/pokemon-service-client/rust-codegen
+
+all: build
+
+codegen:
+	# We need to cd to the top-level git dir otherwise gradle won't run.
```

Comment:
You can use the `--project-dir` Gradle option without needing to `cd`.

Reply:
TIL and thanks